src/index.tsx

```diff
@@ -72,7 +72,7 @@ const advancedOptions: JupyterFrontEndPlugin<Scheduler.IAdvancedOptions> = {
 const telemetryHandler = async (
   eventLog: Scheduler.IEventLog
 ): Promise<void> => {
-  console.log(JSON.stringify(eventLog, undefined, 4));
+  // console.log(JSON.stringify(eventLog, undefined, 4));
 };
 
 const telemetry: JupyterFrontEndPlugin<Scheduler.TelemetryHandler> = {
```

Comment:
Can we merge the definition of `telemetryHandler` and `telemetry` so we can describe both of them in a single doc comment? Below is a rough sketch of my recommendation:

```
// The default telemetry plugin, which does nothing with obtained telemetry. Third-party extensions can override this to collect telemetry data from Jupyter Scheduler by disabling this plugin and exporting their own.
const telemetry: JupyterFrontEndPlugin<Scheduler.TelemetryHandler> = {
  ...
  activate: () => {
    return () => { /* noop */ }
  }
}
```

Open-source users are much more privacy and security conscious, so the word "telemetry" on its own may spook some users. A doc comment to clarify that "this does nothing by default" is good here.

Reply:
Fixed this.